lesson7-questions.txt

```diff
@@ -87,19 +101,26 @@ You will see the SQL that is executed logged to the screen.  What is that SQL?
 There is a security exposure, a risk that a hacker could do damage.  What is that
 risk? What two things are done in your code to prevent the risk?
 
-
+The risk is that its easy for someone to send a delete request directly to the controller, making it possible
+for a user to potentially delete posts they don't own. Something that could be done to mitigate the risk is check to make sure
+a user is logged in before being allowed to do certain actions. A follow up action is to also check if a logged in user is the owner
+of a post.
 
 9. The routes for subscriptions are nested under the forum routes.  How does this
 change the routes?  What is the additional hash key and value that is passed in the
 params hash to the subscriptions controller as a result of this nesting? Which routes
 are unaffected by the nesting?
 
+Since subscriptions are nested under forum routes, they will require the use of /forum routes to be accessed. For example
+"forums/:forum_id/subscriptions". The additional hash key and value passed in the params hash is the forum_id (key) and the id
+of the forum (value). Routes unaffected are ones
```

Comment:
Looks like the answer at the end is incomplete? Which routes are unaffected by the nesting?